docs/index.rst

```diff
@@ -1,11 +1,20 @@
-******************
-hvpy Documentation
-******************
+**********************
+``hvpy`` Documentation
+**********************
 
-This is the documentation for ``hvpy``.
+``hvpy`` is a Python package which provides a high-level interface the `helioviewer.org <https://helioviewer.org>`__ `API <https://api.helioviewer.org/docs/v2/>`__.
 
 .. toctree::
    :maxdepth: 2
 
    guide
    api
+   installation
+   dev_guide
+
+Reporting Issues or Requesting Features
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As mentioned above, building the documentation depends on a number of Sphinx extensions and other packages.
+Since it is not always possible to know which package is causing issues or would need to have a new feature implemented, you can open an issue in the official `hvpy repository <https://github.com/Helioviewer-Project/python-api/issues>`__.
+However, if you wish, you can also open issues in the `helioviewer API repository <https://github.com/Helioviewer-Project/api/issues>`__.
```

Comment:
Technically nothing was mentioned on this page specifically.

Let's keep this simple and say:

```
To report any issues in this documentation, or any problems
found with the API itself, please create an issue on
`our GitHub page <https://github.com/Helioviewer-Project/python-api/issues>`__
```

I think anything found with HvPy should be reported on hvpy and we can transfer the issue to the server API ourselves if that's the problem.